src/praisonai/praisonai/sandbox/e2b.py

```
@@ -0,0 +1,386 @@
+            "running": self._is_running,
+            "api_key_set": bool(os.getenv("E2B_API_KEY")),
+        }
+    
+    async def cleanup(self) -> None:
+        """Clean up sandbox resources."""
+        if self._sandbox:
+            try:
+                # E2B sandbox cleanup is handled by the platform
+                # We can optionally clear files or reset state here
+                pass
+            except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)